astropy/coordinates/baseframe.py

```diff
@@ -30,11 +30,15 @@
                              SphericalRepresentation,
                              UnitSphericalRepresentation,
                              REPRESENTATION_CLASSES)
+from .frame_attributes import FrameAttribute
 
-# Import all FrameAttributes so we don't break backwards-compatibility
+# Import all orher FrameAttributes so we don't break backwards-compatibility
 # (some users rely on them being here, although that is not
 # encouraged, as this is not the public API location.)
-from .frame_attributes import *
+from .frame_attributes import (
```

Comment:
This was partially to keep my is-something-imported checker happy, but mostly because ideally people do *not* import the attributes from `baseframe`. This way, we don't break backwards compatibility, but if we define a new attribute and someone wants to use it, it will break, and they'll realize that they should change...